querybook/webapp/components/QueryReviewsNavigator/QueryReviewsNavigator.tsx

```diff
@@ -8,35 +8,29 @@ import {
     setActiveTab,
     initializeTabFromStorage,
 } from 'redux/queryReview/action';
-import {
-    selectAssignedReviews,
-    selectLoadingAssignedReviews,
-    selectLoadingMyReviews,
-    selectMyReviews,
-} from 'redux/queryReview/selector';
 import { IStoreState } from 'redux/store/types';
 import { Icon } from 'ui/Icon/Icon';
 import { AccentText } from 'ui/StyledText/StyledText';
 import { Tabs } from 'ui/Tabs/Tabs';
-
 import { QueryReviewItem } from './QueryReviewItem';
+import { TooltipDirection } from 'const/tooltip';
+import { Button } from 'ui/Button/Button';
 
 import './QueryReviewsNavigator.scss';
-import { TooltipDirection } from 'const/tooltip';
 
 type TabType = 'myReviews' | 'assigned';
 
 const NAVIGATOR_TABS = [
     {
         key: 'myReviews' as TabType,
-        name: 'Pending Reviews',
+        name: 'Created',
```

Comment:
lets make the naming more consistent, right now we have myReviews and Created mixed together (ReviewType is using CREATED instead of myReviews), let's just stick to one